scripts/build/build.mk

```diff
@@ -44,7 +44,7 @@ build_tags += blst
 build_tags += bls12381
 
 # always include ckzg
-build_tags += ckzg
```

Comment:
@calbera why this change?

Reply:
something I needed to do to get bera-geth building as a dependency. will investigate further